### PR TITLE
SRVCOM-1288 + SRVCOM-1664: Clean up attributes, prereqs

### PR DESCRIPTION
**Pull request by abrennan89**
Jiras: https://issues.redhat.com/browse/SRVCOM-1288 + https://issues.redhat.com/browse/SRVCOM-1664

Applies for OCP 4.6+

Direct previews:
- https://deploy-preview-41860--osdocs.netlify.app/openshift-enterprise/latest/serverless/monitor/serverless-autoscaling-dashboard.html
- https://deploy-preview-41860--osdocs.netlify.app/openshift-enterprise/latest/serverless/security/serverless-custom-domains.html
- https://deploy-preview-41860--osdocs.netlify.app/openshift-enterprise/latest/serverless/monitor/serverless-tracing.html
- https://deploy-preview-41860--osdocs.netlify.app/openshift-enterprise/latest/serverless/monitor/serverless-service-monitoring.html
- https://deploy-preview-41860--osdocs.netlify.app/openshift-enterprise/latest/serverless/security/serverless-ossm-with-kourier-jwt.html
(not much to see, just updating prereqs, minor clean up)

### CODE REVIEW

**Comment by nak3** [comment]
LGTM!

**Comment by jrangelramos** [comment]
QE Approved

File: modules/serverless-autoscaling-dashboard-panic-mode.adoc [hunk 1]
```diff
@@ -1,6 +1,11 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitor/serverless-autoscaling-dashboard.adoc
+
+:_content-type: REFERENCE
 [id="serverless-autoscaling-dashboard-panic-mode_{context}"]
 = Panic mode
 
-The *Panic Mode* graph shows the timeline of times when the Knative service faces a bursty load, which causes the autoscaler to quickly adapt the Service pod number.
+The *Panic Mode* graph shows the timeline of times when the Knative service faces a bursty load, which causes the autoscaler to quickly adapt the service pod number.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/serverless-autoscaling-dashboard-panic-mode.adoc:9]: @nalhadef this PR was already merged yesterday

**Pull request merged**